Add missing data to mini-app.

jamesmcclain opened the pull request:
Missing data have been added to the mini-app.

"Short grass" is of type 81 [1], "chaparral" is of type 52 [2], "desert"
is of type 31 [2], and "tall grass" is of type 85 [1].

To test, it is import to copy the `model.csv` file after the patch has been applied so that the newly-modified version of `model.csv` to the client.  Type:

`vagrant ssh app`

Once there, type:
`cp /opt/app/apps/water_balance/static/data/water_balance/model.csv /var/www/mmw/static/data/water_balance/model.csv`

This copying process is done automatically by the provisioner, but you must copy this file by hand unless you wish to re-provision.

[1] http://www.asprs.org/a/publications/pers/2003journal/december/2003_dec_1387-1397.pdf p. 1389
[2] http://www.mrlc.gov/nlcd11_leg.php

Connects to WikiWatershed/model-my-watershed#185
Connects to WikiWatershed/model-my-watershed#186

rajadain commented on src/mmw/bundle.sh:
```diff
@@ -113,6 +118,7 @@ done
 VAGRANT_COMMAND="cd /opt/app && \
     $ENSURE_DIRS_EXIST && { \
     $COPY_IMAGES_COMMAND &
+    $COPY_SIMPLE_APP_DATA_COMMAND &
```
We actually don't need any changes to `bundle.sh`. Because `model.csv` lives under the `static` directory in a Django app, it is automatically packaged and deployed by the `collectstatic` command:

```bash
./scripts/bundle.sh collectstatic --noinput
```

This command is run during provisioning, thus we don't need the additional commands in `bundle.sh`.

jamesmcclain replied:
That command does not appear to do anything.  I delete /var/www/.../model.csv on the app vm, ran that command, and model.csv has not returned.

rajadain replied:
Sorry, my mistake. The command that I was looking for is

```
./scripts/manage.sh collectstatic --noinput
```

It collects all static assets to the Django static directory. See here for details: https://docs.djangoproject.com/en/1.8/ref/contrib/staticfiles/

kdeloach replied:
Sorry if this is out of context, but I just wanted to point out that the bundle script was designed to generate output directly to `DJANGO_STATIC_ROOT` to eliminate the need to run `collectstatic`.

rajadain commented on src/mmw/js/src/water_balance/models.js:
```diff
@@ -2,20 +2,17 @@
+        '82': ['rowCrops'],
+        '52': ['chaparral'],
+        '31': ['desert'],
+        '85': ['tallGrass']
```
Consider sorting this by ascending NLCD code to ease reading.

rajadain commented:
Well done! The mini app now works perfectly. The following was performed as part of this review:

 * Ran all tests successfully
 * Tried a variety of combinations and they all rendered
 * Height of Water Column stays constant, and correctly represents the component share
 * Numbers are consistent with the "under-the-hood" behavior of the old flash app, although not with the apparent behavior, since the old flash app interpreted the NLCD codes incorrectly as described in #185. This is expected and correct.

jamesmcclain commented:
Okay, I removed the change to `bundle.sh` and changed the testing instructions above.  I will remember to put the correct testing instructions in the commit message when I squash this.

rajadain commented:
Splendid. +1

jamesmcclain commented:
Thank you for the information about the provisioning process, I was unaware.  Thank you for reviewing.
